Resources/Prototypes/Maps/arena.yml

```diff
@@ -65,6 +65,7 @@
             ResearchAssistant: [ 2, 4 ]
             Chaplain: [ 1, 1 ]
             Borg: [ 2, 2 ]
+            Roboticist: [ 1, 1 ]
```

Comment:
```suggestion
```
Remove the Roboticist entry from all maps, these would need to be in individual PRs for maps, since adding a job spawner requires that you touch a map to add it somewhere.